R/run-log.R

```diff
@@ -8,13 +8,9 @@
 #' Future releases will incorporate more diagnostics and parameter estimates, etc. from the runs into this log.
 #' Users can also use `add_config()` to append additional output about the model run.
 #' @seealso `add_config()`
-#' @param .base_dir Directory to search for model YAML files. Only runs with a corresponding YAML will be included.
-#' @param .recurse If `TRUE`, the default, search recursively in subdirectories.
-#' @importFrom stringr str_subset
-#' @importFrom fs dir_ls
-#' @importFrom purrr map map_lgl transpose
-#' @importFrom dplyr as_tibble mutate_at mutate select everything
-#' @importFrom yaml read_yaml
+#' @param .base_dir Directory to search for model YAML files. Only models with a corresponding YAML will be included. Defaults to `get_model_directory()`, and falls back to `getwd()` if `get_model_directory()` returns `NULL`.
+#' @param .recurse If `TRUE`, the default, search recursively in all subdirectories. Passed through to `fs::dir_ls()` -- If a positive number, the number of levels to recurse.
+#' @importFrom purrr map_df
```

Comment:
I like having `run_log`, `config_log`, and `summary_log` documented in their own files because, as mentioned above, it's really more about what comes out of them that we spend time talking about in the docs. And then the ones that have an `add_` variant get that included in their doc.